pkg/kubectl/cmd/create.go

```diff
@@ -52,15 +52,24 @@ var (
 		cat pod.json | kubectl create -f -
 
 		# Edit the data in docker-registry.yaml in JSON using the v1 API format then create the resource using the edited data.
-		kubectl create -f docker-registry.yaml --edit --output-version=v1 -o json`)
+		kubectl create -f docker-registry.yaml --edit --output-version=v1 -o json
+
+		# Create a service using the definition in example-service.yaml.
+		kubectl create -f example-service.yaml
+
+		# Create a replication controller using the definition in example-controller.yaml.
+		kubectl create -f example-controller.yaml
+
+		# Create the objects that are defined in any .yaml, .yml, or .json file within the <directory> directory.
+		kubectl create -f <directory>`)
```

Comment:
Suggest adding `-R` to this command to demonstrate that kubectl supports processing this recursively.